substrate/frame/staking/src/mock.rs

```diff
@@ -570,6 +571,10 @@ impl ExtBuilder {
 		}
 		.assimilate_storage(&mut storage);
 
+		BasicExternalities::execute_with_storage(&mut storage, || {
+			<pallet_session::Pallet<Test> as OnGenesis>::on_genesis();
+		});
+
```

Comment:
You can just run this below inside the `ext` with `execute_with` and then we don't need the new dep.